### PR TITLE
[common] Move policy constants to their own module

### LINKED ISSUE #6299: Add policy for clickhouse replicas to CRDB and Blueprints. At a minimum

### PR DESCRIPTION
**Pull request by andrewjstone**
These constants really don't belong in `address.rs` as they are only tangentially related to addresses.

Also add policy for replicated clickhouse. This will be used in a follow up PR.

Fixes #6299

### CODE REVIEW

File: common/src/policy.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Feet policy related functionality used by both Reconfigurator and RSS.
```
Comment (davepacheco) [inline common/src/policy.rs:5]:
```suggestion
//! Fleet policy related functionality used by both Reconfigurator and RSS.
```

File: sled-agent/src/rack_setup/plan/service.rs [hunk 2]
```diff
@@ -54,11 +56,15 @@ use uuid::Uuid;
 const OXIMETER_COUNT: usize = 1;
 // TODO(https://github.com/oxidecomputer/omicron/issues/732): Remove
 // when Nexus provisions Clickhouse.
-// TODO(https://github.com/oxidecomputer/omicron/issues/4000): Set to 2 once we enable replicated ClickHouse
+// TODO(https://github.com/oxidecomputer/omicron/issues/4000): Use
+// omicron_common::policy::CLICKHOUSE_SERVER_REDUNANCY once we enable
```
Comment (davepacheco) [inline sled-agent/src/rack_setup/plan/service.rs:60]:
```suggestion
// omicron_common::policy::CLICKHOUSE_SERVER_REDUNDANCY once we enable
```

File: sled-agent/src/rack_setup/plan/service.rs [hunk 3]
```diff
@@ -54,11 +56,15 @@ use uuid::Uuid;
...
+// replicated ClickHouse
 const CLICKHOUSE_COUNT: usize = 1;
 // TODO(https://github.com/oxidecomputer/omicron/issues/732): Remove
 // when Nexus provisions Clickhouse keeper.
-// TODO(https://github.com/oxidecomputer/omicron/issues/4000): Set to 3 once we enable replicated ClickHouse
+// TODO(https://github.com/oxidecomputer/omicron/issues/4000): Use
+// omicron_common::policy::CLICKHOUSE_KEEPER_REDUDNANCY once we enable
```
Comment (davepacheco) [inline sled-agent/src/rack_setup/plan/service.rs:66]:
```suggestion
// omicron_common::policy::CLICKHOUSE_KEEPER_REDUNDANCY once we enable
```

**Review by davepacheco — COMMENTED** [review]
I like this!  Just found a few typos -- otherwise it's good.

**Review by davepacheco — APPROVED** [review]

**Comment by karencfv** [comment]
feet policy lol

**Comment by andrewjstone** [comment]
> feet policy lol

Lol. Just about to conk out for the night and noticed this. Off to dream about the American foot liberties union

**Pull request merged**